simc_support/game_data/WowSpec.py

```diff
@@ -370,6 +370,17 @@ def to_dict(self) -> dict:
     full_name="Windwalker",
     simc_name="windwalker",
 )
+MISTWEAVER = WowSpec(
+    id=269,
```

Comment:
Following the table [here](https://wow.tools/dbc/?dbc=chrspecialization&build=9.1.0.38524#page=2) the spec id should be 270.

Reply:
```suggestion
    id=270,
```